File: engine/model/src/main/scala/net/server/channel/packet/alliance/AlliancePlayerRankDataPacket.scala
package net.server.channel.packet.alliance

import net.server.MaplePacket

class AlliancePlayerRankDataPacket( private var _playerId: Int,  private var _rankRaised: Boolean) extends AllianceOperationPacket {
     def playerId: Int = _playerId
     def rankRaised: Boolean = _rankRaised
}

File: engine/engine-model/src/main/java/net/server/channel/packet/alliance/AlliancePlayerRankDataPacket.java
package net.server.channel.packet.alliance;

public class AlliancePlayerRankDataPacket extends AllianceOperationPacket {
   private final int playerId;

   private final boolean rankRaised;

   public AlliancePlayerRankDataPacket(int playerId, boolean rankRaised) {
      this.playerId = playerId;
      this.rankRaised = rankRaised;
   }

   public int playerId() {
      return playerId;
   }

   public boolean rankRaised() {
      return rankRaised;
   }
}
